test/RachisTests/BasicTests.cs

```diff
@@ -90,7 +90,7 @@ public async Task RavenDB_13659()
             using (leader.ContextPool.AllocateOperationContext(out ClusterOperationContext context))
             using (context.OpenWriteTransaction())
             {
-                await mre.WaitAsync();
+                mre.Wait();
```

Comment:
I think it would be better to make this test `void`, so potentially we will not block other tasks

Reply:
done